betterdgg/modules/mentions.js

```diff
@@ -7,9 +7,12 @@
                 gui.lines.on('mousedown', 'div.user-msg a.user', function() {
                     if (bdgg.settings.get('bdgg_highlight_selected_mentions')) {
                         var username = $(this).text();
-                        gui.lines.find(':contains(' + username + ')').addClass('focused');
+                        var users = [$(this).parents().parents().find(".chat-user:contains(" + username + ")")];
```

Comment:
Can this just be?:

``` javascript
var users = gui.lines.find(".chat-user:contains(" + username + ")");
```

(note how I'm not wrapping `users` with `[]`; I don't see why we want this array of user messages to be nested within another array, just for us to only have one element in this array at all times, if that makes sense)